devops/kubernetes/ctrl/app-db-ctrl-job.yaml

```diff
@@ -47,7 +45,133 @@ spec:
         key: node-role.kubernetes.io/master
         operator: Exists
       volumes:
-      - hostPath:
-          path: /opt/openpitrix/pkg/db/schema/app
-          type: ""
+      - configMap:
+          name: openpitrix-app-scheme
         name: flyway
+---
```

Comment:
When db changed, we need to fix two place, it is easy to make mistake. 

How about this:
1. Make a new image which contains db, table and flyway, called openpitrix/openpitrix:flyway
2. Use this new image as a job or initContainer.

Reply:
ok, I will try.